### PR TITLE
chore: upgrade ui-kit

### PR DESCRIPTION
**Pull request by pierrebeitz**
we can make good use of the new table to overhaul the jobs-form

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -152,6 +152,7 @@
     "purify-css": "1.2.5",
     "raml-validator-loader": "0.1.11",
     "react-addons-test-utils": "15.6.2",
+    "react-popper": "2.2.3",
```
Comment (janisz) [inline package.json:155]: Why do we need new dependency?

File: plugins/nodes/src/js/components/NodesTable.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import * as React from "react";
 import {
-  Table,
+  Table_Deprecated,
```
Comment (janisz) [inline plugins/nodes/src/js/components/NodesTable.tsx:3]: I understand table was deprecated in 4.x release
Will you use non deprecated Table in the future?

File: package.json (hunk 1 above)
Comment (pierrebeitz) [reply package.json:155]: it looks like we need to declare this explicitly here as else npm resolves that to an older version that does not use hooks yet and breaks.

File: plugins/nodes/src/js/components/NodesTable.tsx (hunk 2 above)
Comment (pierrebeitz) [reply plugins/nodes/src/js/components/NodesTable.tsx:3]: yes! next PR will have one.

File: package.json (hunk 1 above)
Comment (pierrebeitz) [reply package.json]: we just learned that we had a dependency that relied on popper.js 1.x, which is why i initially had to add this explicitly.

now that dependency is gone. 🎉

**Pull request merged**